paymentsheet/src/test/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreenFormTest.kt

```diff
@@ -1,16 +0,0 @@
-package com.stripe.android.paymentsheet.navigation
-
-import app.cash.turbine.test
-import com.google.common.truth.Truth.assertThat
-import kotlinx.coroutines.test.runTest
-import org.junit.Test
-import org.mockito.kotlin.mock
-
-internal class PaymentSheetScreenFormTest {
```

Comment:
I renamed the screen a few PRs ago, and didn't update the test. Updated tests are there!